http/headers/Link.json

```diff
@@ -34,6 +34,119 @@
+              "safari": {
+                "version_added": false,
+                "impl_url": "https://webkit.org/b/267232"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "expect": {
+          "__compat": {
+            "description": "rel=expect",
+            "spec_url": "https://html.spec.whatwg.org/multipage/links.html#link-type-expect",
+            "support": {
+              "chrome": {
+                "version_added": "124",
+                "impl_url": "https://crbug.com/1507845"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "fetchpriority": {
+          "__compat": {
+            "spec_url": "https://html.spec.whatwg.org/multipage/semantics.html#attr-link-fetchpriority",
+            "tags": [
+              "web-features:fetch-priority"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "103"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "132",
+                "impl_url": "https://bugzil.la/1797715"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "17.2"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
```

Comment:
✅ This is the same (except Chrome 102 -> 103, Firefox 127 -> 132) as:

https://github.com/mdn/browser-compat-data/blob/6295ecf5d57dc8dd197ab3b423b76dca2555768c/api/HTMLLinkElement.json#L262-L314